index.html

```diff
@@ -23,14 +26,47 @@
 <a href="#">Honors</a>
 <a href="#">Contact</a>
 </nav>
-
 </header>
 
 
-<section id = "Home">
+
+<main>  
+
+<section class = "Home" id = "Home">
+
+<div style="width: 5%;">
+
+</div>
+
+<div style="width: 40%;">
+    <img src = "images/personal.png" alt="Nora's Photo" width = "100%" height="auto">
+</div>
+
+<div style="width: 5%;">
```

Comment:
empty divs mustn't be used for spacing.

Reply:
Yes, I removed them before but in another branch